src/pages/join.tsx

```diff
@@ -1,16 +1,19 @@
-import React from 'react';
-import { Header } from '../components/Header/Header';
+import React, { FunctionComponent } from 'react';
+
 import { Button } from '../components/Button/Button';
+import { Footer } from '../components/Footer/Footer';
+import { Header } from '../components/Header/Header';
 import styles from './join.module.scss';
 
-export default function JoinRoute() {
-  return (
-    <>
-      <Header background={true}/>
-      <div className={styles.Join}>
-        <p className={styles.message}>faisons connaissance&nbsp;!</p>
-        <Button>commencer à discuter</Button>
-      </div>
-    </>
-  );
-}
+const JoinPage: FunctionComponent = () => (
+  <>
+    <Header withBackground={true}/>
+    <div className={styles.Join}>
+      <p className={styles.message}>faisons connaissance&nbsp;!</p>
+      <Button>commencer à discuter</Button>
+    </div>
+    <Footer withBackground/>
```

Comment:
This doesn't work 🤔 

<img width="434" alt="Screenshot 2021-01-21 at 20 26 09" src="https://user-images.githubusercontent.com/2527419/105401838-e70f8a00-5c26-11eb-9538-ee5c023f39b5.png">